src/api/dedicated/server/ola/dedicated-server.ola.v6.service.js

```diff
@@ -0,0 +1,24 @@
+angular.module('ovh-api-services').service('OvhApiDedicatedServerOlaV6', ($resource) => {
+  const dedicatedServerOlaResource = $resource('/dedicated/server/:serverName/ola', {
+    serverName: '@serverName',
+  }, {
+    group: {
+      url: '/dedicated/server/:serverName/ola/group',
+      method: 'POST',
+      data: {
```

Comment:
I see `data` for some routes, and `params`for other, there is a reason ? (data doesn't exists for if we refer to documentation : https://docs.angularjs.org/api/ngResource/service/$resource#$resource-arguments)

Reply:
`params` is for URI parameters, while `data` is for body parameters. But maybe the documentation I saw was deprecated, and data is no longer useful... Anyways it works as is and provides a great documentation about what's expected.

Reply:
yes, this is not useful anymore